### DISCUSSION TITLE
Apple Mac 10.13 HS

### DISCUSSION THREAD

**Discussion by ccwking** [rustdesk/rustdesk#6048] [Q&A]
Just installed Rustdesk (Home Use only for remoting to my own computers around the house) and got a question. 

Installed fine on Apple iMac 2017, Intel 64bit.
Installed fine on all PCs (Windows 11)
Wont install on iMac 27 (Late 2008) 32bit. I tried downloading the X86/X64 Mac version and when it opens the folder to drag to applications, its has the no-entry icon. (High Sierra - last official version of Mac OS/X for that iMac)
Think I read it will only install from 10.14, so I suppose there is no way get an compiled version for 10.13 ? 

Chris

**Comment by xlionjuan** [comment]
macOS 10.14 is EOL, and macOS 10.15 is nearly EOL, so maybe you should install Windows 10 on your old Mac instead.

**Comment by Critiqu3** [comment]
this nonsense about being "END OF LIFE"  the computer can do all the real world things that it needs to do...  all of the sudden cant do basic things... i have an old 2012 mac that still runs an old version of chrome remote desktop... 
but yeah... this computer is EOL!  CAUSE  WE ALL HAVE 5 EYEBALLS NOW?!? and 6 ears?!? like our screens arent still 1080p and we arent all still listen to mp3s in 16 bit 44,100k.  watching youtube at 720p 
do we need CHINA to do it befor all of the sudden it is "POSSIBLE" 

i'm sick of all these FAKE GARBAGE OBSTICALS put in place so I CANT GET WORK DONE!  on this overpriced hunk of aluminum.

**Reply by xlionjuan** [reply]
Nobody stopping you to use EOL machines, but you can't force the devs to support EOL OS, again, you should consider Windows 10 LTSC or Linux for your old Mac to use any internet applications, even the software supports, you still shouldn't use EOL OS.
